aiobiketrax/client.py

```diff
@@ -168,11 +170,13 @@ async def update_subscription(self) -> None:
             self._device.unique_id
         )
 
-    async def update_trips(self) -> None:
+    async def update_trip(self) -> None:
```

Comment:
Suggestion to keep this as-is. To be able to specify a from/to, I would suggest to add this to here.